### PR TITLE
feat(): upgrade to angular 9

### PR DESCRIPTION
**Pull request by emoralesb05**
## Description

Upgrade to angular 9 and make sure our build lib process doesnt publish ivy assets.

#### Test Steps

- [ ] `npm install`
- [ ] `npm run serve`
- [ ] Everything component should work fine

#### General Tests for Every PR

- [ ] `npm run serve:prod` still works.
- [ ] `npm run tslint` passes.
- [ ] `npm run stylelint` passes.
- [ ] `npm test` passes and code coverage is not lower.
- [ ] `npm run build:lib` still works.

### CODE REVIEW

File: src/platform/http/actions/http.mixin.spec.ts [hunk 1]
```diff
@@ -4,15 +4,14 @@ import { HttpClientTestingModule, HttpTestingController, TestRequest } from '@an
 import { HttpHeaders, HttpResponse, HttpErrorResponse, HttpParams } from '@angular/common/http';
 import { CovalentHttpModule, TdHttpService } from '@covalent/http';
 import { Observable } from 'rxjs';
-import { TdHttp, TdGET, TdPOST, TdPATCH, TdDELETE, TdParam, TdQueryParams, TdBody } from './';
+import { mixinHttp, TdGET, TdPOST, TdPATCH, TdDELETE, TdParam, TdQueryParams, TdBody } from './';
```
Comment (christianmemije) [inline src/platform/http/actions/http.mixin.spec.ts]: Can we rename `mixinHttp` to something like `TdHttpMixin`?

Comment (emoralesb05) [reply src/platform/http/actions/http.mixin.spec.ts]: We can, but it should be in a separate PR

**Comment by petebacondarwin** [comment]
Great stuff @emoralesb05 and @christianmemije - let me know if you need any further help with Ivy compatibility for v9.

**Comment by emoralesb05** [comment]
Thanks @petebacondarwin ! Gonna merge this so we can start testing it more in our product suite and def will reach out if we encounter any weirdness with the compatibility.

**Pull request merged**